communication.h

```diff
@@ -9,44 +9,44 @@
 #include <errno.h> // Error number definitions
 #include <sys/types.h>
 
-namespace Communication {
+namespace Communication
+{
+enum Commands {
```

Comment:
Die Commands sind meiner Meinung nach fuer die Communication Klasse irrelevant. Diese solltest du in die Bootloader Klasse verschieben